manifests/velero.yaml.tpl

```diff
@@ -44,22 +44,21 @@ spec:
         repository: docker.io/bitnami/kubectl
         tag: "1.29-debian-11"
     configuration:
-      provider: mixed
       namespace: velero
       features: EnableCSI
       defaultBackupTTL: 72h
       defaultResticPruneFrequency: 72h
       backupStorageLocation:
-        name: default
-        provider: community.openstack.org/openstack
-        bucket: ${bucket_velero}
-        config:
-          cloud: self
-          region: ${region}
-          resticRepoPrefix: swift:${bucket_restic}:/restic     
+        - name: default
+          provider: community.openstack.org/openstack
+          bucket: ${bucket_velero}
+          config:
+            cloud: self
+            region: ${region}
+            resticRepoPrefix: swift:${bucket_restic}:/restic     
```

Comment:
we still have restic, does that make sense? same for the other restic mentions in the doc?